core/src/main/resources/lib/layout/search-bar.jelly

```diff
@@ -48,7 +48,7 @@ THE SOFTWARE.
       </div>
       <input value="${attrs.value}"
              id="${attrs.id}"
-             class="jenkins-search__input"
+             class="jenkins-input jenkins-search__input"
```

Comment:
What does this change mean to existing implementations (at least `azure-vm-agents`) that have just the existing class? Do they now need to be adapted, is it wrong to use the previous class without the new class?

Reply:
Good spot, in the case of [azure-vm-agents](https://github.com/jenkinsci/azure-vm-agents-plugin/blob/523ebfeb66cad63b2958c12fdc8309c68643abfc/src/main/resources/com/microsoft/azure/vmagent/AzureVMCloud/new.jelly#L22) it uses a `combobox` which already has a `jenkins-input` class attached. If there are cases (which AFAIK in public repos there aren't) then yeah they'll need the `jenkins-input` class adding.